src/main/java/com/autotune/common/data/system/info/device/accelerator/metadata/AcceleratorProfile.java

```diff
@@ -0,0 +1,41 @@
+package com.autotune.common.data.system.info.device.accelerator.metadata;
+
+public class AcceleratorProfile {
+    private final String profileName;
+    private final double memoryFraction;
+    private final double smFraction;
+    private final int instancesAvailable;
+
+    public AcceleratorProfile(String profileName, double memoryFraction, double smFraction, int instancesAvailable) {
```

Comment:
javadoc here

Reply:
@dinogun Thanks for the review. I have added the java doc